### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by for-ashraf**
In this PR I have incorporated the following:
- Incorporate the details of the Bookstore design in [Zeplin](https://zeplin.io/).
- [Login to Zeplin](https://app.zeplin.io/login) (hint: you will find the credentials in the [sneak peek]- - (https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/sneak_peek_v2_0.md) of the project).
- Open the project Bookstore CMS.
- Use the information detailed in Zeplin to style the website.
- Your final design should match Zeplin's design.

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,76 +1,353 @@
 body {
-  background-color: rgb(250, 250, 250);
+  margin: 0;
 }
 
 * {
+  margin: 0;
   padding: 0;
+}
+
```
Comment (mateo951) [inline src/index.css:11]: Please be sure that your widths are working properly. Im testing it with a width of 1024px and the result is:

#### Your project 
Captura

Also, kindly fix the white background for the form element. As you can see, the color stops at the end of the button and it should keep going as shown in the design 

#### Your project 
image

#### Given Design  
image

File: src/index.css [hunk 2]
```diff
@@ -1,76 +1,353 @@
...
+#root {
```
Comment (mateo951) [inline src/index.css]: It's important to add all elements of the design. Please add the user icon to your project 🚀

image

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @for-ashraf, 

🔥 Highlights 🔥
- Most of the styling is well done 🙌

Great job so far @for-ashraf!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @for-ashraf 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job implementing the requirements by the previous reviewer ✅
- Your repo is professional ✅
- Well structured files ✅
- No linter errors ✅

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I noticed the MIT link in your README is broken, so I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**